.github/workflows/build_deploy.yml

```
@@ -1,7 +1,16 @@
 name: Build
 
 on:
+  push:
+    branches:
+      - [0-9].[0-9]*
```

Should this be:
```suggestion
      - [0-9]+.[0-9]+.*
```

ex: if we had the following branch: `10.10.0`. I guess that won't be a problem for a long time

---

yeah the pattern i chose matches the one we use in github for our branch protection rules. Agreed that a hypothetical 10.10 release would break this.